gnome-extra/cinnamon/cinnamon-4.8.5.ebuild

```diff
@@ -0,0 +1,173 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+PYTHON_REQ_USE="xml"
+
+inherit meson gnome2-utils pax-utils python-single-r1 xdg
+
+DESCRIPTION="A fork of GNOME Shell with layout similar to GNOME 2"
+HOMEPAGE="https://projects.linuxmint.com/cinnamon/"
+SRC_URI="https://github.com/linuxmint/cinnamon/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="+gstreamer gtk-doc +nls +networkmanager"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+KEYWORDS="~amd64 ~arm64 ~x86"
+
+COMMON_DEPEND="${PYTHON_DEPS}
+	app-accessibility/at-spi2-atk:2
+	>=dev-libs/glib-2.52.0:2[dbus]
+	>=dev-libs/gobject-introspection-1.29.15:=
+	dev-libs/libxml2:2
+	>=gnome-extra/cinnamon-desktop-4.8:0=
+	>=gnome-extra/cinnamon-menus-4.8
+	>=gnome-extra/cjs-4.8[cairo]
+	net-libs/libsoup:2.4[introspection]
+	sys-apps/dbus
+	>=sys-auth/polkit-0.100[introspection]
+	virtual/opengl
+	x11-libs/gdk-pixbuf:2[introspection]
+	>=x11-libs/gtk+-3.12.0:3[introspection]
+	>=x11-libs/libnotify-0.7.3:0=[introspection]
+	x11-libs/libX11
+	>=x11-libs/libXfixes-5.0
+	x11-libs/pango[introspection]
+	>=x11-libs/startup-notification-0.11
+	>=x11-wm/muffin-4.8.0[introspection]
+
+	gstreamer? (
+		media-libs/gst-plugins-base:1.0
+		media-libs/gstreamer:1.0
+	)
+	networkmanager? (
+		net-misc/networkmanager[introspection]
+	)
+"
+# caribou used by onscreen keyboard
+# libtimezonemap used by datetime settings
+# iso-flag-png (unpackaged) used by keyboard layout settings
+RDEPEND="${COMMON_DEPEND}
+	>=app-accessibility/caribou-0.3
+	dev-libs/keybinder:3[introspection]
+	dev-libs/libtimezonemap
+	$(python_gen_cond_dep '
+		dev-python/dbus-python[${PYTHON_USEDEP}]
+		dev-python/distro[${PYTHON_USEDEP}]
+		dev-python/pexpect[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pygobject:3[${PYTHON_USEDEP}]
+		dev-python/pyinotify[${PYTHON_USEDEP}]
+		dev-python/pypam[${PYTHON_USEDEP}]
+		dev-python/pytz[${PYTHON_USEDEP}]
+		dev-python/setproctitle[${PYTHON_USEDEP}]
+		dev-python/tinycss2[${PYTHON_USEDEP}]
+		dev-python/xapp[${PYTHON_USEDEP}]
+	')
+	>=gnome-base/dconf-0.4.1
+	>=gnome-base/gsettings-desktop-schemas-2.91.91
+	>=gnome-base/libgnomekbd-2.91.4
+	>=gnome-extra/cinnamon-control-center-4.8[networkmanager=]
+	>=gnome-extra/cinnamon-screensaver-4.8
+	>=gnome-extra/cinnamon-session-4.8
+	>=gnome-extra/cinnamon-settings-daemon-4.8
+	>=gnome-extra/nemo-4.8
+	gnome-extra/polkit-gnome
+	net-misc/wget
+	sys-apps/accountsservice[introspection]
+	sys-power/upower[introspection]
+	>=x11-libs/xapps-2.0.5[introspection]
+	x11-misc/xdg-utils
+	x11-themes/adwaita-icon-theme
+	x11-themes/gnome-themes-standard
+
+	nls? (
+		>=gnome-extra/cinnamon-translations-4.8
+	)
+"
+DEPEND="${COMMON_DEPEND}
+	>=dev-util/intltool-0.40
+	>=sys-devel/gettext-0.17
+	virtual/pkgconfig
+
+	gtk-doc? ( dev-util/gtk-doc )
```

Comment:
These are all `BDEPEND`, makes it looks like you don't need `COMMON_DEPEND` here.